lametro/models.py

```diff
@@ -56,17 +61,21 @@ def api_source(self):
 
     @property
     def api_representation(self):
-        response = requests.get(self.api_source.url)
+        try:
+            response = get_url(self.api_source.url)
+            response.raise_for_status()
```

Comment:
What does `raise_for_status` do for us here?

Reply:
`raise_for_status` raises an `HTTPError` if the request returns a non-200 response. I just added it to keep all the logic in a try/except block.